gcs/retry.go

```diff
@@ -445,7 +451,10 @@ func (rb *retryBucket) CreateObject(
 		fmt.Sprintf("CreateObject(%q)", req.Name),
 		rb.maxSleep,
 		func() (err error) {
-			reqCopy.Contents = bytes.NewReader(contents)
+			if _, err = seeker.Seek(0, io.SeekStart); err != nil {
```

Comment:
nit: use `err :=` here

Reply:
This proposed change produces this error:

```
./retry.go:455:5: err is shadowed during return
```

Should I `return err` as well?  I am trying to follow the named return style used here.

Reply:
Ah, nevermind. Personally, I think naked returns should be avoided, but this code base either disagrees or was largely written before that became a thing :)